package.json

```diff
@@ -27,6 +27,11 @@
       "hash": "QmWY1pHdRP1rA2ifUuCu1ZwFJ8ZzpSEcgXsu9haH21AYKd",
       "name": "go-libp2p-quic-transport",
       "version": "0.2.13"
+    },
+    {
+      "hash": "QmZcLBXKaFe8ND5YHPkJRAwmhJGrVsi1JqDZNyJ4nRK5Mj",
+      "name": "go-multiaddr-net",
```

Comment:
I don't think we need this dependency; what was the rationale?

Reply:
the goal is to be able to dial daemons remotely. for the testbed environment, especially when we're running in a cluster, there's cause to bind to a tcp socket so that a test coordinator can issue commands to remote peers. i think 99% of users would benefit from using a unix socket, however.